src/plugins/content/accessibility/keyNavigation/strategies/itemNavigation.js

```diff
@@ -163,6 +163,9 @@ export default {
                 const choiceType = $choiceInput.attr('type');
 
                 if ($itemElement.hasClass('qti-interaction')) {
+                    //interaction block may be scrollable (if writing-mode for interaction is different from writing-mode for item)
+                    addNavigator($itemElement, $itemElement);
```

Comment:
If it adds a virtual tabstop to all interactions in the item, even if-non scrollable, isn't that quite a big change for keyboard users?

Reply:
I only intended to change choice/extenedText. Fixed.
Then for choice/extenedText it seemed confusing to have 1 tabstop if _interaction=item_ writing mode, and 2 tabstops if _interaction<>item_ writing mode. But maybe that's wrong, changed to check if interaction has `data-scrolling`.

Reply:
Thanks, checking for scrollable or focusable attribute sounds like the better approach.